allennlp/training/trainer.py

```diff
@@ -32,7 +32,8 @@ def __init__(self,
                  cuda_device: int = -1,
                  grad_norm: Optional[float] = None,
                  grad_clipping: Optional[float] = None,
-                 log_one_line_per_batch: bool = False) -> None:
+                 log_one_line_per_batch: bool = False,
+                 validation_metric_decreases: bool = True) -> None:
```

Comment:
ok, that's actually alright, if it always has to begin with that. I'll submit another PR.